testing/e2e/app/routes/Breadcrumbs/spec.ts

```diff
@@ -0,0 +1,94 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.describe('Breadcrumbs', () => {
+  test(`should overflow whenever there is not enough space`, async ({
+    page,
+  }) => {
+    await page.goto(`/Breadcrumbs`);
+
+    const setContainerSize = getSetContainerSize(page);
+    const expectOverflowState = getExpectOverflowState(page);
+
+    await expectOverflowState({
+      expectedItemLength: 5,
+      expectedOverflowButtonVisibleCount: undefined,
+    });
+
+    await setContainerSize('200px');
+
+    await expectOverflowState({
+      expectedItemLength: 2,
+      expectedOverflowButtonVisibleCount: 2,
+    });
+
+    // should restore hidden items when space is available again
+    await setContainerSize(undefined);
+
+    await expectOverflowState({
+      expectedItemLength: 5,
+      expectedOverflowButtonVisibleCount: undefined,
+    });
+  });
+
+  test(`should at minimum always show one overflow tag and one item`, async ({
+    page,
+  }) => {
+    await page.goto(`/Breadcrumbs`);
+
+    const setContainerSize = getSetContainerSize(page);
+    const expectOverflowState = getExpectOverflowState(page);
+
+    await expectOverflowState({
+      expectedItemLength: 5,
+      expectedOverflowButtonVisibleCount: undefined,
+    });
+
+    await setContainerSize('10px');
+
+    await expectOverflowState({
+      expectedItemLength: 1,
+      expectedOverflowButtonVisibleCount: 1,
+    });
+  });
+});
+
+// ----------------------------------------------------------------------------
+
+const getSetContainerSize = (page: Page) => {
+  return async (dimension: string | undefined) => {
+    await page.getByTestId('container').evaluate(
+      (element, args) => {
+        if (args.dimension != null) {
+          element.style.setProperty('width', args.dimension);
+        } else {
+          element.style.removeProperty('width');
+        }
+      },
+      { dimension },
```

Comment:
nit: `dimension` makes it sound like the possible values would be `horizontal` or `vertical`. It might be better to name this `size` or even just `value`